flake.nix

```diff
@@ -117,9 +117,8 @@
         import ./checks { inherit inputs system pkgs; }
       );
 
-      # TODO change this to something that has better looking output rules
-      # Nix formatter available through 'nix fmt' https://nix-community.github.io/nixpkgs-fmt
-      formatter = forAllSystems (system: nixpkgs.legacyPackages.${system}.nixfmt-classic);
+      # Nix formatter available through 'nixfmt' https://nix-community.github.io/nixpkgs-fmt
+      formatter = forAllSystems (system: nixpkgs.legacyPackages.${system}.nixfmt);
```

Comment:
I believe it is currently not recommended to use the `nixfmt` package while we're migrating to the new rfc style nixfmt. Currently `nixfmt` *is* the same as `nixfmt-classic` but may switch to `nixfmt-rfc-style` at some point, with little warning. I wasn't sure which of the two you would rather use, so I went with `nixfmt-classic` as it is more similar to `nixpkgs-fmt`.
If I know which to use, I can clean up this PR to use one or the other from the beginning.

Reply:
Okay, good to know. Let's go with nixfmt-rfc-style then since it will be the way going forward.